input/pagecontent/reference.md

```diff
@@ -872,13 +872,13 @@ For a path to a code within a code system, use this syntax:
 
 **Examples:**
 
-* The set the to the designation use of the code `#active` in the [Condition Clinical Status value set](https://terminology.hl7.org/ValueSet-condition-clinical.html):
+* To set the designation.use of the code `#active`:
```

Comment:
Any reason to cut out the reference to the value set where #active is defined in the context of a code hierarchy? I thought that was useful because the reader can see the original VS.

Reply:
This rule would have to be on a `CodeSystem`, so I thought it was confusing to reference a value set. We could instead reference the code system: https://terminology.hl7.org/CodeSystem-condition-clinical.html, but then I feel like it is a bit confusing, why are you redefining a code system that already exists?

Reply:
That's a good point!